server/views/pages/event.njk

```diff
@@ -1,8 +1,14 @@
 {% extends 'layout/default.njk' %}
 
 {% block pageMeta %}
-  {% component 'open-graph', { pageConfig: pageConfig, article: event } %}
-  {% component 'twitter-card', { pageConfig: pageConfig, article: event } %}
+  {% set metaContent = {
+    type: 'wellcome:event' + '.' + event.format,
```

Comment:
After discussion with @jennpb we're going to use 'website' for all `og:type` attributes that aren't an 'article'.